src/utils/prices.ts

```diff
@@ -97,6 +98,11 @@ export function warningSeverity(priceImpact: Percent | undefined): 0 | 1 | 2 | 3
   return 0
 }
 
+export function warningFiatSeverity(priceImpact: Percent | undefined): 0 | 1 {
+  if (!priceImpact?.lessThan(ALLOWED_FIAT_PRICE_IMPACT_HIGH)) return 1
```

Comment:
We should thrive for consistency. `warningSeverity` returns `3` when price impact is high. 

`ALLOWED_FIAT_PRICE_IMPACT_HIGH` should map to 3 or be changed to `_LOW`

Reply:
Think that we should keep the name `_HIGH` as the intention is to display high risk. Mapping to `3` in this one.